wagmi.ts

```diff
@@ -0,0 +1,35 @@
+import { getDefaultWallets } from '@rainbow-me/rainbowkit'
+import { chain, configureChains, createClient } from 'wagmi'
+import { publicProvider } from 'wagmi/providers/public'
+import { alchemyProvider } from 'wagmi/providers/alchemy'
+
+const { chains, provider } = configureChains(
+ [chain.mainnet, chain.optimism, chain.arbitrum],
+ [alchemyProvider({ apiKey: 'RbvJ2z-poT2yB-cpZ9xUMaauzUeTQVf5' })],
```

Comment:
this api key should be an environment variable. we should generate a new key to use for production, and maybe another for local / testing

Reply:
Ok - awesome. Yes was thinking we could have one for the frontend and backend and also for staging and production as well!